src/lib/output.test.ts

```
@@ -0,0 +1,214 @@
+  });
+
+  it('does not quote simple alphanumeric values', () => {
+    const logs: string[] = [];
+    const origLog = console.log;
+    console.log = (msg: string) => logs.push(msg);
+
+    try {
+      setToonMode(true);
+      outputTable(['Name'], [['Alice']]);
+      // Tabular TOON: header on first line, values on second line starting with 2 spaces
+      expect(logs[0]).toMatch(/\n  Alice\n?$/);
```

ESLint

[error] Spaces are hard to count. Use {2}.

(no-regex-spaces)